ecs-cli/modules/cli/compose/project/project_parseV3.go

```diff
@@ -51,24 +74,92 @@ func (p *ecsProject) parseV3() (*[]containerconfig.ContainerConfig, error) {
 		return nil, err
 	}
 
-	// convert ServiceConfigs to ContainerConfigs
-	conConfigs := []containerconfig.ContainerConfig{}
-	for _, service := range config.Services {
-		cCon, err := convertToContainerConfig(service)
+	return config, nil
+}
+
+func convertToContainerConfig(serviceConfig types.ServiceConfig) (*containerconfig.ContainerConfig, error) {
+	//TODO: Add Healthcheck to ContainerConfig
+	c := &containerconfig.ContainerConfig{
+		CapAdd:  serviceConfig.CapAdd,
+		CapDrop: serviceConfig.CapDrop,
+		// Devices:               serviceConfig.Devices
+		DockerSecurityOptions: serviceConfig.SecurityOpt,
+		Name:             serviceConfig.Name,
+		Image:            serviceConfig.Image,
+		Hostname:         serviceConfig.Hostname,
+		Links:            serviceConfig.Links,
+		Privileged:       serviceConfig.Privileged,
+		ReadOnly:         serviceConfig.ReadOnly,
+		Command:          serviceConfig.Command,
+		User:             serviceConfig.User,
+		WorkingDirectory: serviceConfig.WorkingDir,
+	}
+
+	if serviceConfig.DNS != nil {
+		c.DNSServers = serviceConfig.DNS
+	}
+	if serviceConfig.DNSSearch != nil {
+		c.DNSSearchDomains = serviceConfig.DNSSearch
+	}
+	if serviceConfig.Entrypoint != nil {
+		c.Entrypoint = serviceConfig.Entrypoint
+	}
+	if serviceConfig.ExtraHosts != nil {
+		extraHosts, err := utils.ConvertToExtraHosts(serviceConfig.ExtraHosts)
 		if err != nil {
 			return nil, err
 		}
-		conConfigs = append(conConfigs, *cCon)
+		c.ExtraHosts = extraHosts
 	}
+	if serviceConfig.Tmpfs != nil {
+		tmpfs := yaml.Stringorslice(serviceConfig.Tmpfs)
+		ecsTmpfs, err := utils.ConvertToTmpfs(tmpfs)
+		if err != nil {
+			return nil, err
+		}
+		c.Tmpfs = ecsTmpfs
+	}
+	if serviceConfig.Logging != nil {
```

Comment:
Looks like this is doing what `ConvertToLogging` in `convert_task_definition.go` is doing --noting for posterity that that method could probably be refactored to take a string (Logging.Driver) and string Map (Logging.Options) to be more generic/reusable for both v3 and v1/v2 conversion.